app/helpers/application_helper.rb

```diff
@@ -1,4 +1,9 @@
 module ApplicationHelper
+
+  def user_signed_in?
+    session[:user_id].present?
+  end
+
   def display_picture(game)
     if game.picture.nil?
```

Comment:
This is not needed.  It is already designated as a helper method in application controller with the line "helper_method :user_signed_in?"